### DISCUSSION TITLE
Student account was terminated unexpectedly and nowhere to renew

### DISCUSSION THREAD

**Discussion by WangMinan** [community/community#53366] [GitHub Education] [answered]
I just finished my first year with my student account of github and successfully reverify my account weeks ago.
However, I received **a email** this morning which asked me to **reverify** my academic status like this.
![image](https://user-images.githubusercontent.com/77133573/233524149-d3f6ab4a-3131-40c2-98fc-53324ed9a83e.png)
When I tried to turn to https://education.github.com/benefits as required, it directly redirect me to https://education.github.com/globalcampus/student which is the homepage of global-campus. 
Meanwhile I can't find the **submit your information again** button as mentioned in the mail.
So I clicked the **reverify your academic affilication** button here.
![image](https://user-images.githubusercontent.com/77133573/233524598-dd025b40-48c5-46ca-948f-bbd90f4e8e1a.png)
But it said that it is not available for me to reverify.
![image](https://user-images.githubusercontent.com/77133573/233524753-d1f12f03-cd9e-4d34-9468-ac752de43cbd.png)

I wonder how can I get reverified. Thanks a lot. 😢

**Comment by WangMinan** [comment]
I suggest that this might be a bug.

**Reply by ElliotQi** [reply]
Have you solved this? I'm meeting this problem too.

**Reply by alumik** [reply]
> Have you solved this? I'm meeting this problem too.

When I checked today, I discovered that the student benefits and GitHub Pro coupons have reappeared. This came as a pleasant surprise, and I suspect that it might be due to a recent update or bug fix on the GitHub platform.

**Comment by alumik** [comment]
I have encountered a similar experience whereupon I initiated a support ticket several days ago; however, I have yet to receive a response.

**Reply by alumik** [reply]
To my dismay, I discovered today that my ticket was closed by Github, and I was instructed to seek assistance through community inquiries.

**Reply by WangMinan** [reply]
Same reply for my ticket.
![AD5FB4C4-92C5-4382-AC69-3C115D8533E8](https://user-images.githubusercontent.com/77133573/235576872-67ff0626-69b0-4033-b069-e470beb4351e.png)
Still have no idea how to solve this.

**Reply by WangMinan** [reply]
@samus-aran Sorry to bother you. I find that many answers were offered by you in Github Education section and would appreciate it if you could help us with the situation above.  I really don’t know how to solve this.

**Comment by Lossita** [comment]
got the same issue but no one can answer me😢

**Comment by WangMinan** [comment, marked as answer]
![image](https://github.com/community/community/assets/77133573/43172397-a15c-46d2-bd95-e0d9f89955fe)
![image](https://github.com/community/community/assets/77133573/f5b009a8-edea-4d05-9e99-6c59434e411b)
It seems that this problem has been fixed.
